frontend/components/global/Wakelock.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+    <div v-if="wakeIsSupported" class="d-print-none d-flex px-2"
+        :class="$vuetify.breakpoint.smAndDown ? 'justify-center' : 'justify-end'">
+        <v-switch v-model="wakeLock" small :label="$t('recipe.screen-awake')" />
+    </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from "@nuxtjs/composition-api";
+import { computed, onMounted, onUnmounted } from "@nuxtjs/composition-api";
+import { useWakeLock } from "@vueuse/core";
+
+export default defineComponent({
+    setup(props, context) {
```

Comment:
You can leave this empty if you're not using props/context

```suggestion
    setup() {
```